torchvision/csrc/io/decoder/audio_stream.cpp

```diff
@@ -7,13 +7,13 @@ namespace ffmpeg {
 
 namespace {
 bool operator==(const AudioFormat& x, const AVFrame& y) {
-  return x.samples == y.sample_rate && x.channels == y.channels &&
+  return static_cast<int>(x.samples) == y.sample_rate && static_cast<int>(x.channels) == y.channels &&
```

Comment:
This line causes clang_format error: https://app.circleci.com/pipelines/github/pytorch/vision/8395/workflows/8f202b72-937b-4c98-b642-1e679e41b5c7/jobs/604228, could you fix that please?